### ISSUE TITLE
Auth Proxy (keycloak, authelia,..)

### ISSUE THREAD

**Issue by Tuphal** [SergiX44/XBackBone#375]
**Is your feature request related to a problem? Please describe.**
I use Authelia as SSO for my services. An implementation of a Auth Proxy would be great for SSO.

**Describe the solution you'd like**
Proxy Auth implementation like grafana, grocy, Tandoor Recipes, calibre web,...

**Comment by EricGuic** [comment]
I agree, I also plan to use Authelia ("inside" SWAG I use as a reverse proxy), and being able to "link" XBackbone and Authelia would be perfect !

**Comment by Pfuenzle** [comment]
Hi, any updates on this? Would be great to see this
